src/cr/cube/subtotal.py

```diff
@@ -21,7 +21,10 @@ def is_valid(self):
             required_keys = {'anchor', 'args', 'function', 'name'}
             has_keys = set(self._data.keys()) == required_keys
             if has_keys:
-                return self._data['function'] == 'subtotal'
+                is_subtotal = self._data['function'] == 'subtotal'
```

Comment:
If you move the `subtotal` function check into the `if` above, you can avoid doing work of the 2 lines below when it's false.

Reply:
:+1: